electron/renderer/src/lib/__mocks__/localStorage.ts

```diff
@@ -18,6 +18,8 @@
  */
 
 export class LocalStorageMock {
+  store: {[key: string]: string} = {};
+
   constructor() {
     this.store = {};
   }
```

Comment:
I guess you don't need the constructor now since you initialize the value in the class
```suggestion

```